dotcom-rendering/src/components/ShareButton.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import { css } from '@emotion/react';
+import { ArticleDesign, ArticleDisplay, Pillar } from '@guardian/libs';
+import type { StoryProps } from '../../.storybook/decorators/splitThemeDecorator';
+import {
+	defaultFormats,
+	splitTheme,
+} from '../../.storybook/decorators/splitThemeDecorator';
+import { palette as themePalette } from '../palette';
+import {
+	CopyLinkButton,
+	NativeShareButton,
+	ShareButton,
+} from './ShareButton.importable';
+
+export default {
+	component: ShareButton,
+	title: 'Components/ShareButton',
+};
+
+interface StoryArgs extends StoryProps {
+	theme: string;
+}
+
+export const CopyLink = () => {
+	return (
+		<ShareButton
+			pageId={'123'}
+			webTitle={'The the'}
+			format={{
+				display: ArticleDisplay.Standard,
+				theme: Pillar.News,
+				design: ArticleDesign.Standard,
+			}}
+			context="ArticleMeta"
+		/>
+	);
+};
+CopyLink.storyName = 'CopyLink';
+CopyLink.decorators = [splitTheme()];
+
+export const CopyLinkXSmall = () => {
+	return (
+		<ShareButton
+			pageId={'123'}
+			webTitle={'The the'}
+			size="xsmall"
+			format={{
+				display: ArticleDisplay.Standard,
+				theme: Pillar.News,
+				design: ArticleDesign.Standard,
+			}}
+			context="LiveBlock"
+		/>
+	);
+};
+CopyLinkXSmall.storyName = 'CopyLinkXSmall';
```

Comment:
Same point as above:

```suggestion
export const ShareButtonXSmallStory = () => {
	return (
		<ShareButton
			pageId={'123'}
			webTitle={'The the'}
			size="xsmall"
			format={{
				display: ArticleDisplay.Standard,
				theme: Pillar.News,
				design: ArticleDesign.Standard,
			}}
			context="LiveBlock"
		/>
	);
};
CopyLinkXSmall.storyName = 'ShareButtonXSmall';
```